.github/workflows/deploy.yml

```diff
@@ -0,0 +1,79 @@
+name: Build and publish
```

Comment:
Let's have a more meaningful name and eventually write down a couple of comments to better explain the workflow